src/api/api_dvr.c

```diff
@@ -531,6 +554,8 @@ void api_dvr_init ( void )
     { "dvr/entry/rerecord/allow",  ACCESS_RECORDER, api_dvr_entry_rerecord_allow, NULL },
     { "dvr/entry/stop",            ACCESS_RECORDER, api_dvr_entry_stop, NULL },   /* Stop active recording gracefully */
     { "dvr/entry/cancel",          ACCESS_RECORDER, api_dvr_entry_cancel, NULL }, /* Cancel scheduled or active recording */
+    { "dvr/entry/previouslyrecorded", ACCESS_RECORDER, api_dvr_entry_previouslyrecorded, NULL }, /* Add selected schedules as previously recorded */
```

Comment:
I think this is similar to #1020 (where you press "stop" on upcoming recordings to stop them from recording)?

Reply:
I'm sorry I haven't seen that pull request before, I'm come from argustv server with a pc, and in argus I had the option to select one program and mark it as previously recorded and then the autorec rule will skip it in the time, so I tried to replicate it in tvheadend, in this case the button only is enabled if you select a upcoming recording and disable if the program is recording now.

When you press it popup a question to be sure to add it as previously recorded and then move it to "removed recordings". So the autorecord rule still active and will skip same episode or program in the time.